client/dialog/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title></title>
+    <link id="favicon" href="/favicon.png" rel="icon" type="image/png" />
+    <link href="/dialog/style.css" rel="stylesheet" />
+    <script>
+      window.addEventListener(
+        "message",
```

Comment:
Is this the only message that the dialog will handle now and into the future?
If the message will eventually include a function code, perhaps that should be established now?

Reply:
This PR is just replacing the existing `wiki.dialog` behaviour of displaying some static content in a dialog window.